docs/csharp/language-reference/operators/member-access-operators.md

```diff
@@ -143,9 +152,37 @@ You also use parentheses to adjust the order in which to evaluate operations in
 
 [Cast expressions](type-testing-and-cast.md#cast-operator-), which perform explicit type conversions, also use parentheses.
 
+## Index operator ^
```

Comment:
@BillWagner what is the "official" name of the `^` operator? In the compiler error (when I set language version to 7.3), it's called "index operator". In [LDM from a year ago](https://github.com/dotnet/csharplang/blob/master/meetings/2018/LDM-2018-09-05.md#index-operator) and in the [Roslyn code](https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/Portable/Operations/UnaryOperatorKind.cs#L48), it's called "hat operator".

I've chosen the "index" variant to match the public interface of the compiler error message.

Reply:
That's a good question.

ping @jcouv @agocke for the official name.

Reply:
From the proposal:

> We call this (`^`) the *index from end* operator.

Reply:
@BillWagner thanks; I've updated the text and links. That looks to be the last unresolved question, so this PR is ready for your final review.